apps/performance-tests/src/unified-selection/SelectionScope.test.ts

```diff
@@ -0,0 +1,166 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { expect } from "chai";
+import { SnapshotDb } from "@itwin/core-backend";
+import { createECSqlQueryExecutor } from "@itwin/presentation-core-interop";
+import { computeSelection } from "@itwin/unified-selection";
+import { ComputeSelectionProps } from "@itwin/unified-selection/lib/cjs/unified-selection/SelectionScope";
+import { Datasets, IModelName } from "../util/Datasets";
+import { run, RunOptions } from "../util/TestUtilities";
+
+describe("compute", () => {
+  const additionalElementCount = 5;
+
+  function createElementKeys({ start, count }: { start: number; count: number }) {
```

Comment:
Looks like you're creating ids here rather than keys

Reply:
Renamed.